app/views/shared/_subnavigation.html.erb

```diff
@@ -46,7 +46,7 @@
       <li>
         <%= layout_menu_link_to t("layouts.header.sdg"),
                                 sdg_goals_path,
-                                controller_path.split("/").first == "sdg",
+                                controller_path == "sdg/goals" && action_name != "help",
```

Comment:
Could you expand on the reasons for this change? Should there be an active section in the subnavigation when we're in the SDG help page? If so, why? If not, why not? :smile: